clients/admin-ui/src/pages/classify-systems/index.tsx

```diff
@@ -56,13 +56,15 @@ const ClassifySystems: NextPage = () => {
     }
   }, [dispatch, allSystems]);
 
+  // Poll for updates to classification until all classifications are finished
+  const [shouldPoll, setShouldPoll] = useState(true);
   const { isLoading: isLoadingClassifications, data: classifications } =
     useGetAllClassifyInstancesQuery(
       {
         resource_type: GenerateTypes.SYSTEMS,
         fides_keys: systems?.map((s) => s.fides_key),
       },
-      { skip: !hasPlus }
+      { skip: !hasPlus, pollingInterval: shouldPoll ? 20 * 1000 : undefined }
```

Comment:
This works great — thanks @allisonking. Should we make the polling interval shorter?

Reply:
Sure, I changed it to 5 seconds